### PR TITLE
feat: have separate  routes instead of sections

### LINKED ISSUE #331: Improve layout to have tabs instead of anchors
Clicking on the menu should open in different tabs, that would result in a shorter page.

### PR DESCRIPTION
**Pull request by LinaYahya**
closes #331

### CODE REVIEW

File: src/components/layout/Sidebar.tsx [hunk 1]
```diff
@@ -8,41 +9,50 @@ import PersonIcon from '@mui/icons-material/Person';
...
+  const { itemId } = useParams();
+  const navigate = useNavigate();
+  const { pathname } = useLocation();
   const { descendantApps } = useContext(DataContext);
 
   return (
     <GraaspMainMenu>
       <MenuItem
-        onClick={() => scrollTo('general')}
+        onClick={() => navigate(buildItemPath(itemId))}
```
Comment (spaenleh) [inline src/components/layout/Sidebar.tsx:32]: Can we support CMD + click on these links ? To open in a new tab ?

File: src/components/pages/Item/AppsAnalyticPage.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const AppsAnalyticPage = (): JSX.Element | null => {
+  const { t } = useAnalyticsTranslation();
+  const { descendantApps } = useContext(DataContext);
+
+  return descendantApps.length > 0 ? (
+    <Box id={APPS_ID} paddingTop={2}>
+      <SectionTitle title={t('APPS_ANALYTICS_TITLE')} />
+      <AppsAnalytics />
+    </Box>
+  ) : null;
```
Comment (spaenleh) [inline src/components/pages/Item/AppsAnalyticPage.tsx:20]: use if and else to make it more clear ?

**Review by spaenleh — APPROVED** [review]
Looks good ! Just a suggestion for links and a nitpick on readability.

**Comment by pyphilia** [comment]
So good, I was waiting for so long to have that <3

**Pull request merged**